tasks/create_target_vm/02_engine_vm_configuration.yml

```diff
@@ -21,6 +21,7 @@
     register: db_vm_update
     with_items:
       - {field: 'origin', value: 6}
+      - {field: 'custom_bios_type', value: 2}
```

Comment:
Please add a comment here or both values. Direct IDs can be very confusing.
Actually...
no, this is not a good place/way, you should pass it as a regular parameter when creating VM. 'origin' is here only because we don't have API for it

Reply:
so that's what I tried (that was the reason for [the PR to ansible-collection](https://github.com/oVirt/ovirt-ansible-collection/pull/111) but when trying to set it here I get:
```
20:01:39 [ ERROR ] fatal: [localhost]: FAILED! => {"changed": false, "msg": "Unsupported parameters for (ovirt_vm) module: bios_type Supported parameters include: affinity_group_mappings, affinity_label_mappings, allow_partial_import, auth, ballooning_enabled, boot_devices, boot_menu, cd_iso, clone, clone_permissions, cloud_init, cloud_init_nics, cloud_init_persist, cluster, cluster_mappings, comment, cpu_cores, cpu_mode, cpu_pinning, cpu_shares, cpu_sockets, cpu_threads, custom_compatibility_version, custom_properties, delete_protected, description, disk_format, disks, domain_mappings, exclusive, export_domain, export_ova, fetch_nested, force, force_migrate, graphical_console, high_availability, high_availability_priority, host, host_devices, id, initrd_path, instance_type, io_threads, kernel_params, kernel_params_persist, kernel_path, kvm, lease, lun_mappings, memory, memory_guaranteed, memory_max, migrate, name, nested_attributes, next_run, nics, numa_nodes, numa_tune_mode, operating_system, placement_policy, poll_interval, quota_id, reassign_bad_macs, rng_device, role_mappings, serial_console, serial_policy, serial_policy_value, smartcard_enabled, snapshot_name, snapshot_vm, soundcard_enabled, sso, state, stateless, storage_domain, sysprep, template, template_version, ticket, timeout, timezone, type, usb_support, use_latest_template_version, vmware, vnic_profile_mappings, wait, watchdog, xen"}
```
I suspect that the reason is that this project doesn't consume the ansible-collection yet, so I took Evgeny's approach in #331  - not ideal, but seems like a workaround we can have until we're able to set it as the other properties

Reply:
ok. it is going to be less ugly with a clear TODO comment there

Reply:
added